### PR TITLE
Fix index.d.ts in @types/react-text-mask

### PR DESCRIPTION
**Pull request by kvrvgixzis**
mask is required. if mask undefined throwing error "Cannot read property 'length' of undefined"
maskArray isnt bool, but is it cant be true, only false

### CODE REVIEW

**Comment by kvrvgixzis** [comment]
Playground

https://codesandbox.io/s/falling-sun-xdthu?file=/src/App.js

File: types/react-text-mask/index.d.ts [hunk 1]
```diff
@@ -8,11 +8,11 @@
 
 import * as React from "react";
 
-export type maskArray = Array<string | RegExp> | boolean;
+export type maskArray = Array<string | RegExp> | false;
 
 export interface MaskedInputProps
     extends React.InputHTMLAttributes<HTMLInputElement> {
-    mask?: maskArray | ((value: string) => maskArray);
+    mask: maskArray | ((value: string) => maskArray);
```
Comment (amcasey) [inline types/react-text-mask/index.d.ts:15]: It would be nice to add tests showing that omitting the property or passing `true` results in an error.

**Review by samarmohan — CHANGES_REQUESTED** [review]
Thanks for contributing!

add tests like @amcasey said, otherwise 👍🏽.

**Review by samarmohan — CHANGES_REQUESTED** [review]
tests please

**Comment by kvrvgixzis** [comment]
how write test example if types now rights? tests failes. revert to old types?

**Comment by kvrvgixzis** [comment]
Sorry, it's my first PR in public repo

**Comment by samarmohan** [comment]
You did an amazing job! The tests are good and all checks passed! Congrats!!!!!

**Review by samarmohan — APPROVED** [review]

**Review by cavarzan — APPROVED** [review]

**Comment by kvrvgixzis** [comment]
Ready to merge

**Pull request merged**